<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>544</y>
     <width>301</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>180</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border:none;
}

QPushButton:hover{
	background-color:rgb(207, 31, 213)
}
</string>
   </property>
   <widget class="QPushButton" name="btn_prev">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/front-64.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_directory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/add.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/back-64.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/pause-96.png)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/mode-50.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_volume">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/voice-150.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="list_music">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">*{background-color: rgba(0,0,0,0);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSlider" name="music_slider">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>530</y>
     <width>354</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lab_pos">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>510</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Chakma</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_total">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>510</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>21</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>40</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Chakma</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_bk">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>640</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>640</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>360</width>
     <height>640</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lab_bk</zorder>
  <zorder>widget</zorder>
  <zorder>list_music</zorder>
  <zorder>music_slider</zorder>
  <zorder>lab_pos</zorder>
  <zorder>lab_total</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
